app/serializers/host_serializer.rb

```diff
@@ -11,7 +11,10 @@ class HostSerializer
   end
 
   has_many :test_results
+
+  # rubocop:disable Style/SymbolProc
```

Comment:
Unfortunately if the symbol proc `&:all_profiles` is used, the framework passes some options to it as an argument.

Reply:
Yeah, I couldn't figure out how to use symbol to proc here either :(

Reply:
If the method have had at least one argument, then it would work. But I'm not sure what's the logic (why) behind the framework sending some `options` to it.